ftdetect/eex.vim

```diff
@@ -0,0 +1,2 @@
+au BufRead,BufNewFile *.eex set filetype=eex
```

Comment:
We're trying to get rid of `ftdetect` rules in this plugin... #2259

Reply:
Ah ok, according to this comment I need to contribute upstream to Vim, then have that come to Neovim via a patch?

https://github.com/nvim-treesitter/nvim-treesitter/pull/2268#issuecomment-1013878699

Reply:
That would be ideal, yes. (Such a trivial change usually gets merged quickly.)

Reply:
In any case, if you drop that file from the PR, we could merge it now. (You can tell your users to put the autocommand in their `init.vim` in the meantime.)

Reply:
@clason I removed the `ftdetect` rules as requested. Is there anything else you'd like to see in this PR?